src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
@HeahDude Well, for the configuration of services the users would not have to do that. It's the responsibility of us to set up the parser properly in the `YamlFileLoader`. So we are more talking about someone who uses the parser in their own code.